packages/syria/README.md

```diff
@@ -0,0 +1,38 @@
+# SY Syria
+
+Contains all the regions of Syria:
```

Comment:
I think they are called "governorates" or "provinces" or "counties" in English (source: https://en.wikipedia.org/wiki/Governorates_of_Syria)